src/main/scala/viper/gobra/frontend/Config.scala

```diff
@@ -241,6 +247,8 @@ case class Config(
     } else {
       TypeBounds(Int = TypeBounds.IntWith64Bit, UInt = TypeBounds.UIntWith64Bit)
     }
+
+  lazy val backendOrDefault: ViperBackend = backend.getOrElse(ConfigDefaults.DefaultBackend)
```

Comment:
does this need to be lazy? If so, we should probably add a comment. Otherwise, I would drop the "lazy", given that this seems to always be read anyway.

In addition, if this needs to be lazy and given how "easy" the computations are and how infrequently we read this, I would probably make this a `def` instead, but this is a minor point

Reply:
I don't see any necessity for making it lazy. I'm changing it to be a regular `val`